ui/component/publishAdditionalOptions/view.jsx

```diff
@@ -57,6 +56,10 @@ function PublishAdditionalOptions(props: Props) {
     setHideSection(!hideSection);
   }
 
+  React.useEffect(() => {
+    if (disabled) setHideSection(disabled);
+  }, [disabled]);
```

Comment:
Since the commit message mentioned "hide additional options on reload" rather than "hide additional options when 'disabled' changes to 'true'", the following might be a more 'correct' way to express the intent:

```
  React.useEffect(() => {
    setHideSection(false);
  }, []);
```
(Empty dependancy = run 1 time when mounted)

But at that point, doing the following would be enough:
```
const [hideSection, setHideSection] = React.useState(true);
```
(start as hidden on mount/reload).

But I would assume we would want the expansion state to be persistent -- vaguely recall that was a complaint -- some wanted expanded, some don't.  I have no preference either way, just commenting here.

Reply:
But if the intent is really "hide additional options when 'disabled' changes to 'true'", then nothing to change.
This would mean the section collapse but never expands back when you switch between Video and Post, for example.

Reply:
Thanks! Well, the reason why the persistent state can become an issue here is because the timestamp for the date gets set once you open that section. In other words, if you open up the publish form and just leave it open for a while until you actually upload a file you might end up releasing the file in the past so to speak. I guess I can just use both variables in combination here. Use persistent value if the form is not disabled...

Reply:
>_ In other words, if you open up the publish form and just leave it open for a while until you actually upload a file you might end up releasing the file in the past so to speak_

It sounds like a bug on the Date widget instead (maybe my mistake in the past).  It shouldn't be locked to a date when the section opens, but should always be "default" on a new form, which the 'action' side would then take `Date.now()` when calling the API.

Reply:
```
const [date, setDate] = React.useState(releaseTime ? linuxTimestampToDate(releaseTime) : new Date());
```
I think `new Date()` is wrong and should be `undefined`. But need to test to confirm the api is called with latest time.

Reply:
Aight, let's put this PR on hold.

Reply:
Placed the fix in `fix.release.date` branch -- tested to match production's payload.  Feel free to cherry-pick.  

Turns out, it was just a GUI bug -- while it is showing a locked date, it is still `undefined` in Redux unless you actually changed the value. 

----

I noticed another issue with the calendar widget (also happening in production, not sure when broken) -- every time I change a date, the value goes to year 202, and the saved value is negative.